### PR TITLE
Alert text updated for empty Address on Send

### PR DESCRIPTION
**Pull request by Rockindash**
Alert text updated for empty Address on Send, Issue #1255

### CODE REVIEW

**Comment by johnnyluo** [comment]
need to rebase it

File: VultisigApp/VultisigApp/Localizables/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -397,3 +397,6 @@
 "saveVaultQR" = "Save Vault QR";
 "joinSend" = "Join Send";
 "joinSwap" = "Join Swap";
+"invalidAddress" = "Invalid Address";
+"invalidAddress" = "Invalid Address";
```
Comment (johnnyluo) [inline VultisigApp/VultisigApp/Localizables/en.lproj/Localizable.strings:401]: These two are exactly the same

**Review by johnnyluo — APPROVED** [review]
LGTM

File: VultisigApp/VultisigApp/Localizables/en.lproj/Localizable.strings (hunk 1 above)
Comment (Rockindash) [reply VultisigApp/VultisigApp/Localizables/en.lproj/Localizable.strings]: Just fixed this and the conflict.

**Pull request merged**